docs/sources/trackers/javascript-trackers/web-tracker/cross-domain-tracking/index.md

```diff
@@ -4,23 +4,351 @@ date: "2022-08-30"
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  useExtendedCrossDomainLinker: true,
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+newTracker('namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  useExtendedCrossDomainLinker: true,
+});
+```
+
+  </TabItem>
+</Tabs>
+
+To include specific properties:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('newTracker', 'namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  useExtendedCrossDomainLinker: {
+    userId: true;
+    sessionId: true;
+    sourceId: true;
+    sourcePlatform: true;
+    reason: true; // can also be a callback
+  },
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+newTracker('namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  useExtendedCrossDomainLinker: {
+    userId: true;
+    sessionId: true;
+    sourceId: true;
+    sourcePlatform: true;
+    reason: true; // can also be a callback
+  },
+});
+```
+
+  </TabItem>
+</Tabs>
+
+Provide a callback to `reason` in the form `((evt: Event) => string)` to add custom information to the querystring.
+
+
+## Short format
+
+Provide a callback for the `crossDomainLinker` option to configure which links to decorate with the short `_sp` format. It should be a function taking one argument, the link element, that returns `true` if the tracker should decorate the link element and `false` otherwise.
+
+### Decorate some links
+
+For example, this function would only decorate those links whose destination is `http://acme.de/` or whose HTML ID is `crossDomainLink`:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+snowplow('newTracker', 'namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function (linkElement) {
+    return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+  }
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+newTracker('namespace', '{{collector_url_here}}', {
+  appId: 'my-app-id',
+  appVersion: '0.1.0',
+  cookieSameSite: 'Lax',
+  crossDomainLinker: function (linkElement) {
+    return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+  }
+});
+```
+
+  </TabItem>
+</Tabs>
+
+If you want to decorate every link to the domain `github.com`:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return /^https:\/\/github\.com/.test(linkElement.href);
+  }
+}
+```
+
+You can also call the `crossDomainLinker` function directly:
```

Comment:
~This probably needs an explainer; you would want to do this because it follows the old link tracking behaviour where it only adds the event listener to decorate the link to links that already exist on the page. If you're in an SPA or have some other dynamic content, those links won't be decorated even if they would satisfy your callback.~

~So it's probably something you'd want to do after each `trackPageView` call to make everything up to date.~

Oh, saw the below section, this should just be removed and left to the "Update event listeners" section.